include/libopencm3/stm32/l4/memorymap.h

```diff
@@ -56,7 +56,10 @@
 #define I2C1_BASE			(PERIPH_BASE_APB1 + 0x5400)
 #define I2C2_BASE			(PERIPH_BASE_APB1 + 0x5800)
 #define I2C3_BASE			(PERIPH_BASE_APB1 + 0x5c00)
+#define CRS_BASE			(PERIPH_BASE_APB1 + 0x6000)
 #define CAN1_BASE			(PERIPH_BASE_APB1 + 0x6400)
+#define USB_DEV_FS_BASE			(PERIPH_BASE_APB1 + 0x6800)
+#define USB_PMA_BASE			(PERIPH_BASE_APB1 + 0x6c00)
```

Comment:
Minor nitpick: fix tab spacing

Reply:
i think it is... checkout the raw file... am i missing something???

Reply:
Oh you're right, for some reason it was showing up wrong in the diff. Sorry! Hope this gets merged soon!